src/App.tsx

```diff
@@ -1,36 +1,20 @@
-import { SettingsIcon } from '@chakra-ui/icons';
-import {
-  Box,
-  ChakraProvider,
-  Tab,
-  TabList,
-  TabPanel,
-  TabPanels,
-  Tabs,
-} from '@chakra-ui/react';
+import { Box, ChakraProvider } from '@chakra-ui/react';
 import { useInterpret, useSelector } from '@xstate/react';
-import { useEffect, useMemo } from 'react';
-import { ActorsPanel } from './ActorsPanel';
+import { useEffect } from 'react';
 import { AuthProvider } from './authContext';
 import { authMachine } from './authMachine';
 import './base.scss';
 import { CanvasProvider } from './CanvasContext';
-import { CanvasPanel } from './CanvasPanel';
-import { toDirectedGraph } from './directedGraph';
-import { EditorPanel } from './EditorPanel';
-import { EventsPanel } from './EventsPanel';
+import { CanvasView } from './CanvasView';
+import { Footer } from './Footer';
```

Comment:
This PR doesnt compile at the moment because this file doesnt exist anymore.
```suggestion
import { Footer } from './Footer';
```